spec/index.html

```diff
@@ -324,39 +325,46 @@ <h3>Shared blank nodes, unions and merges</h3>
       to identify a blank node, but unless these documents are in a shared identifier scope
       or are derived from a common source,
       the occurrences of "<code>_:x</code>" in one document
-      will identify a different blank node than the one in the graph described by the other document.
+      will identify a different blank node than the occurrences of "<code>_:x</code>"
+      in the graph described by the other document.
       When graphs are formed by combining RDF from multiple sources,
       it may be necessary to <dfn class="no-export lint-ignore">standardize</dfn> apart the blank node identifiers
-      by replacing them by others which do not occur in the other document(s).
-      For example, the two graphs represented  by the following texts:</p>
+      by replacing them with other identifiers which do not occur in the other document(s).
+      For example, the two graphs represented by the two texts below contain two nodes each, 
```

Comment:
```suggestion
      For example, the two graphs for the two texts below contain two nodes each, 
```

Reply:
It is better to use a vague relationship than to have different, not-supported-by-semantics relationships.

Reply:
Fine